### PR TITLE
Created a requirement.txt file

### PR DESCRIPTION
**Pull request by Ycandido0119**
### Asana Ticket

Cleaning up git files
Installing debugging tools

### PR Description

I set up a requirement file and added all the packages to install needed in order to work on this project. I also worked using ipdb to debug some of the code in the `my_custom_commands.py` and made some changes to that file. Finally made a migrations as well.

### Checklist

[ x ] Went through and checked that I haven't committed anything I shouldn't.
[ x ] I have gone over the commit with a mentor.
[ x ] I have understood the logic of the ticket and if not wrote questions on things I am stuck on.

### CODE REVIEW

File: octoapp/challenges/management/commands/my_custom_command.py
Comment (chloe-umble) [inline octoapp/challenges/management/commands/my_custom_command.py:1]: I think it makes sense to remove these management command files from this PR and merge them separately.

File: octoapp/challenges/migrations/0004_remove_reading_meter_meterregister_and_more.py
Comment (chloe-umble) [inline octoapp/challenges/migrations/0004_remove_reading_meter_meterregister_and_more.py:1]: This migration should also go in a separate PR with the `my_custom_command` change

**Review by chloe-umble — CHANGES_REQUESTED** [review]
The README and requirements.txt looks good! But the changes to the management command should be merged separately in a different PR

File: octoapp/challenges/migrations/0004_remove_reading_meter_meterregister_and_more.py
Comment (Ycandido0119) [reply octoapp/challenges/migrations/0004_remove_reading_meter_meterregister_and_more.py:1]: So run git revert and then only push the changes where I added the requirement file and made changes to the README.

**Review by chloe-umble — APPROVED** [review]
